CreatePackages.md

```diff
@@ -240,4 +240,4 @@ You can also log into chocolatey.org and upload your package from there (not rec
 Yes - [[AutomaticPackages]]
 
 ## Becoming a primary maintainer of an existing package
-See [[PackageMantainerHandover]]
\ No newline at end of file
+See [[PackageMantainerHandover]]
```

Comment:
I used the GitHub in browser editor. I assume that it just added the "newline at end of file".